### DISCUSSION TITLE
Asus g15 drains more battery on eco mode

### DISCUSSION THREAD

**Discussion by ghost** [seerge/g-helper#1108] [Q&A]
With Ghelper when I use eco mode and my gpu gets turned off it starts using more battery than it is with standard. Why is this?

**Comment by cosmicAP** [comment]
same thing i happens to me too. whenever i switch to eco mode my discharge level goes up , best to keep it in optimized mode. also mind if you can tell windows power mode helps  in battery life . i set it to balanced on my tuf dash f 15 and i don't see any difference in battery life.

**Reply by ghost** [reply]
For me it feels like something that shouldn't happen but it could be some kind of program from asus that doesn't get deleted when you uninstall AC or some bios setting maybe

**Reply by cosmicAP** [reply]
could be the case best thing to do is to keep it in optimized mode.

**Comment by utchka** [comment]
Asus Rog Strix G17 2023
My laptop drains 60W in standard mode with iGPU active, 50W in ultimate mode with rtx4060 active.
(plugged, balanced power plan, idling on desktop)

Shoudn't it be the opposite? 

Not G-Helper fault anyway, I have the same results by disabling it.

**Reply by ghost** [reply]
maybe an asus program or bios setting?

**Comment by cosmicAP** [comment]
> Asus Rog Strix G17 2023 My laptop drains 60W in standard mode with iGPU active, 50W in ultimate mode with rtx4060 active. (plugged, balanced power plan, idling on desktop)
> 
> Shoudn't it be the opposite?
> 
> Not G-Helper fault anyway, I have the same results by disabling it.

50 Watt much be your RTX's tgp. although sitting ideal with igpu on and pulling 60 watt seems odd

**Reply by utchka** [reply]
50W is the total power used by the laptop measured with an energy meter connected to the power supply. 
When playing games it goes up to 130W. 
In comparison, my desktop pc drains almost 100W when idling and 350W when playing. 
But I still don't get why switching to dGPU decreases the power drain of my laptop.

**Reply by ghost** [reply]
that's what I'm thinking but I can't grasp what would be doing this, maybe an asus program or bios setting?

**Reply by utchka** [reply]
It could be some hidden bios setting. I've noticed a cpu temperature increase after updating the bios to the latest version (324). Unfortunately Asus does not write anything about bios releases, so we don't know what changes are made.

**Comment by utchka** [comment]
On battery:

![iGPU](https://github.com/seerge/g-helper/assets/139227466/372f9f96-d8d1-4634-a745-311c5fc8b924)
(iGPU - 36W)

![dGPU](https://github.com/seerge/g-helper/assets/139227466/163915df-53ee-46f6-a44d-1e6033ea4d3a)
(dGPU - 33.8W)

**Reply by cosmicAP** [reply]
is that is the discharge rate ? if yes and on slient mode the it is bad. my advice is to not use laptop in battery mode as the it will reduce the battery life . my laptop hovers between 5-20 watt max on battery (silent mode).

**Reply by utchka** [reply]
Wow. So your laptop lasts many hours on battery mode, I guess. Mine lasts a couple of hours if I use it for browsing and reading/writing emails.

**Comment by seerge** [comment]
Hello guys, 

From what I have noticed G15 seem to have a notoriously bugged BIOS. And it could be that in your case(s) BIOS just doesn't power off dGPU (but reports that it did, therefore AC/G-Helper would show it as eco)

I even mentioned this in FAQ https://github.com/seerge/g-helper#i-have-g15-2022-and-my-gpu-refuses-to-disableenable-or-my-fans-misbehave you can try to roll back to BIOS 311 or 315 it seem to behave OK

Look at this issue https://github.com/seerge/g-helper/issues/878#issuecomment-1684878602 as well, and experience of a user from it

**Reply by utchka** [reply]
Hi seerge, I can't flash my bios to a previous version, it doesn't let me! I've tried both method, from the bios itself and from windows.
It gives me an error in the first case, and in the second case windows just reboots normally and nothing happens.
I've tried version 316 and 319 downloaded from asus official website. Now I have  324. Also, I can re-flash 324 no problem, but as I said it doesn't accept older versions :-(

**Reply by seerge** [reply]
@utchka well, my point is that it's could be an issue with BIOS in a first place. 

You may check this comment btw https://github.com/seerge/g-helper/discussions/1042#discussioncomment-6765972 - apparently for someone having "AC control interface" enabled in BIOS itself has made a difference

**Reply by utchka** [reply]
Unfortunately it didn't work for me. Thanks anyway.
I just want to update you about my laptop behavior. Eco and Ultimate mode actually work, the problem comes with Standard and Optimized. The dGPU stays active in both Standard and Optimized along with the iGPU, which translates in about 15W of wasted energy and a temp. increase of 4-5°C for the CPU.

**Reply by seerge** [reply]
Optimized is just an automated switch between Eco and Standard:) nothing else
